tests/test_tree.py

```diff
@@ -71,13 +71,26 @@ def test_unserializable_tree(self):
         tree = self.manager.tree
 
         item = tree.root_item.create_item("item.a", "Item A", "Item A")
-        # Store a datetime object as this can't be serialized.
-        item.properties["dateteime"] = datetime.datetime.now()
+        # Store a set object as this can't be serialized.
+        item.properties["set"] = set()
 
         # Check that we get the error we expect.
-        with self.assertRaisesRegex(TypeError, "datetime.* is not JSON serializable"):
+        with self.assertRaisesRegex(TypeError, "set is not JSON serializable"):
             tree.save(StringIO())
 
+    def test_serialize_tree_with_datetime_property(self):
+        """
+        Ensures we can save a tree when an item property is a datetime object
+        """
+
+        # Create some items that will modify
+        tree = self.manager.tree
+
+        item = tree.root_item.create_item("item.a", "Item A", "Item A")
+        item.properties["datetime"] = datetime.datetime.now()
+
+        tree.save(StringIO())
```

Comment:
Could you load the information back and check that you get a datetime object again?